src/relayserver/RelayServer.js

```diff
@@ -286,7 +288,11 @@ class RelayServer extends EventEmitter {
         console.error('web3 subscription:', error)
       }
     }).on('data', this._workerSemaphore.bind(this)).on('error', (e) => { console.error('worker:', e) })
-    setTimeout(() => { this.web3.eth.getBlockNumber().then(blockNumber => this._workerSemaphore.bind(this)({ number: blockNumber })) }, 1)
+    const handler = () => { this.web3.eth.getBlockNumber().then(blockNumber => this._workerSemaphore.bind(this)({ number: blockNumber })) }
```

Comment:
over-clutter the logs. 
run handler only if `blockNumber!= this.lastScannedBlock`

Reply:
This is temporary. Shahaf will take over and fix these issues